test/src/unit-vfs.cc

```diff
@@ -464,31 +442,26 @@ TEMPLATE_LIST_TEST_CASE("VFS: File I/O", "[vfs][uri][file_io]", AllBackends) {
 
   // Write to two files
   URI largefile = URI(path.to_string() + "largefile");
-  require_tiledb_ok(vfs.write(largefile, write_buffer, buffer_size));
+  REQUIRE_NOTHROW(vfs.write(largefile, write_buffer, buffer_size));
   URI smallfile = URI(path.to_string() + "smallfile");
-  require_tiledb_ok(
-      vfs.write(smallfile, write_buffer_small, buffer_size_small));
+  REQUIRE_NOTHROW(vfs.write(smallfile, write_buffer_small, buffer_size_small));
 
   // On non-local systems, before flushing, the files do not exist
   if (!(path.is_file())) {
-    require_tiledb_ok(vfs.is_file(largefile, &exists));
-    CHECK(!exists);
-    require_tiledb_ok(vfs.is_file(smallfile, &exists));
-    CHECK(!exists);
+    CHECK(!vfs.is_file(largefile));
+    CHECK(!vfs.is_file(smallfile));
```

Comment:
Catch2 [discourages](https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#natural-expressions) writing `CHECK(!foo)`, providing `CHECK_FALSE(foo)` in its place.

There are too many instances in our tests that you don't have to do anything now; will open a story later.